src/Field.php

```diff
@@ -28,7 +28,7 @@ class Field
      *
      * @var string
      */
-    protected $key;
+    protected $key = '';
```

Comment:
We don't need a default value for this variable.

Reply:
Correct.